tests/test_ppx.ml

```diff
@@ -0,0 +1,15 @@
+let f0 = [%mysql Select_one "SELECT TRUE"]
```

Comment:
Could these functions be named in a better way? Because now it is difficult from the expect test to know what the function is expected to do.

Reply:
Fair enough. I'll come up with more descriptive names.